sample-script/watchResource.js

```diff
@@ -0,0 +1,87 @@
+const { BotkitConversation } = require("botkit");
+
+module.exports = function (controller) {
+    let stop;
+
+    controller.hears(new RegExp(/((watch)$)|((watch) [a-zA-Z]*$)/), "direct_mention", async (bot, message) => {
+        var trimmedMessage = message.text.replace("watch ", "");
+        if (stop != undefined) {
+            console.log("Watch has already been started!");
+
+            await bot.reply(message, {
+                blocks: [
+                    {
+                        type: "section",
+                        text: {
+                            type: "mrkdwn",
+                            text: "Uh oh! You already have started a watch!",
+                        },
+                    },
+                ],
+            });
+            return;
+        }
+
+        await bot.reply(message, {
+            blocks: [
+                {
+                    type: "section",
+                    text: {
+                        type: "mrkdwn",
+                        text: `Kubernetes watch running on ${trimmedMessage}!`,
+                    },
+                },
+            ],
+        });
+
+        const kubeResource = {
+            type: trimmedMessage,
+            resource: "",
+            group: "",
+        };
+        stop = controller.plugins.kubernetes.startWatch(
```

Comment:
the fact that this is assigned to a single instance variable `stop` implies that you can only have one watch going at any given time - is that how this was intended to work?  or am I missing something here?

Reply:
We addressed this during demo that triggering multiple watches can be handled by the bot on startup. This sample-script feature is intended to showcase how the plugin can be triggered, however lab-partner will handle this for users rather than rely on input.